asyncpgsa/connection.py

```diff
@@ -15,32 +15,41 @@
 _dialect.paramstyle = 'named'
 
 
+def replace_keys(keys, querystring, params, inline=False):
+    new_query = querystring
+    new_params = []
+    for key, value in enumerate(keys):
```

Comment:
this is misleading, we are enumerating a list called 'keys', calling the key 'value' and the index 'key', that is super confusing. If we instead just enumerate `params` we could do it like this
```
for index, param in enumerate(params)
    var_name, value = param  #unpack
```